### DISCUSSION TITLE
My display rate is lower than osu

### DISCUSSION THREAD

**Discussion by YourLocalIndonesianArtist** [ppy/osu#16946] [Q&A]
Yeah so whrn i was messin around with things in the settings i accidentically changed the display settings to a higher one soo... hei do i change the display settings without actually usin the app

**Comment by bdach** [comment]
I don't understand what this discussion is trying to say. Can you explain properly?

**Reply by YourLocalIndonesianArtist** [reply]
okay so this was a long time ago but i still have the same issue. i cant open osu!lazer because of the display rate of the game is higher than my  monitor, i think it was because i was messing around with my settings in osu!lazer it changed the display rate in the game to 67.5 or more, so the answer im looking for is how to actually lower the display rate of the game without opening it.

simplified how to reduce display rate without opening the game itself.

**Reply by peppy** [reply]
https://github.com/ppy/osu/search?q=overclocked&type=issues

**Reply by YourLocalIndonesianArtist** [reply]
ah it seems that i have the answer for it myself, so all i did was make the game borderless or windowed and it works as it should be. 
i appreciate the effort to help me out, thank you. :)
